index.bs

```diff
@@ -208,14 +202,23 @@ spec: webidl
   <section>
     <h3 id="reading-current-states">Reading the current permission state</h3>
     <p>
-      |descriptor|'s <dfn export>permission state</dfn> is one of {{"granted"}},
-      {{"prompt"}}, or {{"denied"}}, indicating respectively if the calling
-      algorithm should succeed without prompting the user, show the user a
-      prompt to decide whether to succeed, or fail without prompting the user.
-      The UA must return whichever of these values most accurately reflects the
-      user's intent. Subsequent uses of |descriptor|'s <a>permission state</a>
-      with the same <a>current settings object</a> must return the same value,
-      unless the UA receives <a>new information about the user's intent</a>.
+      |descriptor|'s <dfn export lt="permission state|state">permission state
+        </dfn> is one of {{"granted"}}, {{"prompt"}}, or {{"denied"}},
+      indicating respectively if the calling algorithm should:
+    </p>
+
+    <ul>
+      <li>succeed without prompting the user ({{"granted"}}),</li>
+      <li>show the user a prompt to decide whether to succeed ({{"prompt"}}),</li>
+      <li>or fail without prompting the user ({{"denied"}}).</li>
+    </ul>
+
+    <p>
+      The UA returns whichever of these values most accurately reflects the user's
```

Comment:
I intended these paragraphs to be the algorithm that determines "|descriptor|'s permission state". Obviously that wasn't clear enough. Should we say "|descriptor|'s permission state is the result of the following algorithm"?